docs/setup/changing-the-colors.md

```diff
@@ -208,12 +208,12 @@ The following fields must be set for each toggle:
   [9]: #accent-color
   [10]: https://github.com/squidfunk/mkdocs-material/tree/master/material/.icons
 
-### System preference
+### Inferring the initial setting from system preference
```

Comment:
We shouldn't change the headline title, because we'd lose all anchor links. Please change it back.